src/main/java/ay2021s1_cs2103_w16_3/finesse/model/transaction/Date.java

```diff
@@ -7,16 +7,20 @@
 import java.time.LocalDate;
 import java.time.format.DateTimeFormatter;
 import java.time.format.DateTimeParseException;
+import java.time.format.ResolverStyle;
 
 /**
  * Represents a Transaction's date in the finance tracker.
  * Guarantees: immutable; is valid as declared in {@link #isValidDate(String, Clock)}
  */
 public class Date implements Comparable<Date> {
 
-    public static final String MESSAGE_CONSTRAINTS = "Dates should be of the format dd/mm/yyyy "
-            + "and cannot be later than the current date";
-    public static final DateTimeFormatter VALIDATION_FORMAT = DateTimeFormatter.ofPattern("dd/MM/yyyy");
+    public static final String EPOCH_STRING = "01/01/1970";
+    public static final String MESSAGE_CONSTRAINTS = "Dates should be of the format dd/mm/yyyy, "
```

Comment:
```suggestion
    public static final String MESSAGE_CONSTRAINTS = "Dates should be a valid calendar date of the format dd/mm/yyyy, "
```

Just in case someone enters `30/02/2020` and files a bug report stating that the error message is not informative?

Reply:
Ok. Should this change be mirrored in the user guide?

Reply:
That would be good I think. We can properly define what a valid calendar date is in the user guide.